pkg/controller/perconaservermongodb/psmdb_controller.go

```diff
@@ -541,6 +543,7 @@ func (r *ReconcilePerconaServerMongoDB) safeDownscale(cr *api.PerconaServerMongo
 		// downscale 1 pod on each reconciliation
 		if *sf.Spec.Replicas-rs.Size > 1 {
 			rs.Size = *sf.Spec.Replicas - 1
+			fmt.Println("TEST: rs", rs.Name, rs.Size)
```

Comment:
please remove debug

Reply:
fixed in d63633e678f9b40252449f0704914f4cde22b716